data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg

```diff
@@ -446,13 +446,13 @@
 
                 [message]
                     speaker=Hamel
-                    message= _ "Bah! They look like fat slugs to me. Come on boys, let’s chop them to pieces before they can leave the canal."
+                    message= _ "Bah! They look like fat slugs to me. Come on, boys, let’s chop them to pieces before they can leave the canal!"
```

Comment:
The exclamation mark might be good, but the extra comma isn't.

Reply:
OK.